### DISCUSSION TITLE
Migrate into  Postgres Database multiple Schema

### DISCUSSION THREAD

**Discussion by tapsu01** [prisma/prisma#5971] [Q&A] [answered]
Is there a way to run a migration and have it execute inside a postgres multiple SCHEMA

**Comment by ryands17** [comment, marked as answer]
@tapsu01 👋 

You will be able to add tables to other schemas by simply editing the `.sql` file created during migration, but it will not be visible in your `schema.prisma`.

As for querying those, you should be able to do so with [raw queries](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) and you will get the resulting data. For now, Prisma only supports a single schema in your `schema.prisma`.

It would be great if you could share your use case for multiple schemas [here](https://github.com/prisma/prisma/issues/2443) :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
